ansys-grantami-serverapi-openapi/poetry.lock

```diff
@@ -373,21 +373,21 @@ yaml = ["ruamel.yaml"]
 
 [[package]]
 name = "pytest"
-version = "8.0.0"
+version = "7.4.4"
```

Comment:
Any reason for downgrading pytest?

Reply:
The template defines pytest as ^7.0.0, this needs to be bumped to ^8.0.0 to allow us to use a later version. I'll make this change, release a new template, and update this PR.

Reply:
https://github.com/ansys/openapi-client-template/pull/152